android/src/main/kotlin/live/hms/hmssdk_flutter/HMSPeerExtension.kt

```diff
@@ -39,11 +39,6 @@ class HMSPeerExtension {
                 HMSPeerUpdate.ROLE_CHANGED -> "roleUpdated"
                 HMSPeerUpdate.METADATA_CHANGED -> "metadataChanged"
                 HMSPeerUpdate.NAME_CHANGED -> "nameChanged"
-                HMSPeerUpdate.BECAME_DOMINANT_SPEAKER -> "becameDominantSpeaker"
```

Comment:
`type == HMSPeerUpdate.BECAME_DOMINANT_SPEAKER || type == HMSPeerUpdate.NO_DOMINANT_SPEAKER`

these checks are not to be removed